src/components/AccountTokensTable.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <table>
+    <thead>
+      <tr>
+        <th>
+          Symbol
+          <hint-tooltip>
+            {{ tokensHints.tokenSymbol }}
+          </hint-tooltip>
+        </th>
+        <th>
+          Name
+          <hint-tooltip>
+            {{ tokensHints.tokenName }}
+          </hint-tooltip>
+        </th>
+        <th>
+          Contract
+          <hint-tooltip>
+            {{ tokensHints.smartContractId }}
+          </hint-tooltip>
+        </th>
+        <th>
+          Amount
+          <hint-tooltip>
+            {{ tokensHints.amount }}
+          </hint-tooltip>
+        </th>
+        <th>
+          Value
+          <hint-tooltip>
+            {{ tokensHints.value }}
+          </hint-tooltip>
+        </th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr
+        v-for="token in accountTokens?.data"
+        :key="token.contractId">
+        <td>
+          <app-link :to="`/tokens/${token.contractId}`">
+            {{ token.tokenSymbol }}
+          </app-link>
+        </td>
+        <td>
+          <app-link :to="`/tokens/${token.contractId}`">
+            {{ token.tokenName }}
+          </app-link>
+        </td>
+        <td>
+          <app-link :to="`/contracts/${token.contractId}`">
+            {{ token.contractId }}
+          </app-link>
+        </td>
+        <td>
+          {{ token.amount }}
+        </td>
+        <td>
+          {{ token.value !== null ? `$${formatNumber(token.value)}` : 'N/A' }}
```

Comment:
Could this be?
```suggestion
          {{ !!token.value ? `$${formatNumber(token.value)}` : 'N/A' }}
```

Reply:
I'm afraid not null is very important here, otherwise 0$ will be displayed as N/A